src/modules/site-settings/edit-site-details.tsx

```diff
@@ -101,7 +101,7 @@ export default function EditSiteDetails( { currentWpVersion, onSave }: EditSiteD
 				} catch ( wpError ) {
 					console.error( 'Error changing WordPress version:', wpError );
 					const errorMessage = stripAnsi( ( wpError as Error )?.message );
-					setIsChangeWpError( __( 'Error changing WordPress version.' ) );
+					setIsChangeWpError( __( 'Error changing WordPress version' ) );
```

Comment:
We finish in a dot in path error. I guess it's expected as it has two sentences.

![Screenshot 2025-03-20 at 09 58 21](https://github.com/user-attachments/assets/21a2b0b5-2797-44a3-a7a3-0dfd18fc58e4)

Reply:
Thanks for sharing the context. I tested it again and noticed we display this error two times: in dialog and as a form validation error. Technically, it's not a form validation error, but a form submission error. I've removed the form validation error, so the dialog is the only place that reports the error.